### PR TITLE
Add HINT that users can reset the rate and pitch on track load

### PR DESCRIPTION
**Pull request by J-Kappes**
Added HINT that users can reset the rate and pitch on track load.

### CODE REVIEW

**Comment by JoergAtGithub** [comment]
I wonder, why there are no CI runs for this PR?

**Comment by ronso0** [comment]
yeah, that's strange.
@J-Kappes Do you mind rebasing you branch and force-push?
and while you're at it, please reword the commit message to "add HINT that users can reset the rate and pitch on track load" (since that is more meaningful than the current message ; )
Thanks!

**Comment by J-Kappes** [comment]
> I wonder, why there are no CI runs for this PR?

Maybe because I started this as a draft, idk

**Comment by Swiftb0y** [comment]
> I wonder, why there are no CI runs for this PR?

IIUC that can happen when the most recent commit is a clean merge commit or if the checks already failed in the feature branch in the repo of the PR author...

File: source/chapters/user_interface.rst [hunk 1]
```diff
@@ -602,6 +602,10 @@ control rate changes also from your computer's keyboard, see the chapter
   the Pitch/Rate slider positioned at the center, the pitch rate display is at
   +0.0%, which indicates no change.
 
+.. hint:: To make the rate and/or pitch automatically reset whenever a new
+          track is loaded go to :menuselection:`Preferences --> Decks -->
+          Speed (Tempo) and Key (Pitch) options --> Reset on track load`.
+
```
Comment (ronso0) [inline source/chapters/user_interface.rst:608]: the indentation has to match the above paragraph = +2 spaces

**Review by ronso0 — CHANGES_REQUESTED** [review]

**Comment by ronso0** [comment]
pre-commit fails with
```
fatal: could not read Username for 'https://gitlab.com/': No such device or address
```
:shrug:

**Comment by JoergAtGithub** [comment]
The fix for the gitlab issue is in https://github.com/mixxxdj/manual/pull/529, but for some reasons this PR fails on CI.

**Comment by daschuer** [comment]
The Pioneer link is fixed here: https://github.com/mixxxdj/manual/pull/532 the https://web.archive.org links are still OK, probably a temporary issue.

File: source/chapters/user_interface.rst (hunk 1 above)
Comment (ronso0) [reply source/chapters/user_interface.rst:608]: oh, sorry, my hint was unclear obviously.
Indentation inside the hint block was fine, just the entire block should be at the same i. level as the paragraph above.

Comment (ronso0) [reply source/chapters/user_interface.rst:608]: actually, pre-commit would notice that if it wouldn't fail earlier.

**Comment by JoergAtGithub** [comment]
If you merge in the changes from 2.3 branch, the Sphinx-Linkcheck fails in CI should be solved.

**Review by ronso0 — APPROVED** [review]

**Comment by ronso0** [comment]
@JoergAtGithub wanna press merge : )

**Comment by JoergAtGithub** [comment]
Pressed!

**Pull request merged**